jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/profiles/ProfileWatcher.kt

```diff
@@ -4,35 +4,45 @@
 package software.aws.toolkits.jetbrains.core.credentials.profiles
 
 import com.intellij.openapi.Disposable
+import com.intellij.openapi.application.ApplicationManager
 import com.intellij.openapi.util.Disposer
-import com.intellij.openapi.util.io.FileUtil
 import com.intellij.openapi.vfs.AsyncFileListener
 import com.intellij.openapi.vfs.LocalFileSystem
-import com.intellij.openapi.vfs.VfsUtilCore
 import com.intellij.openapi.vfs.VirtualFileManager
 import com.intellij.openapi.vfs.newvfs.events.VFileEvent
+import com.intellij.openapi.vfs.pointers.VirtualFilePointer
 import software.amazon.awssdk.profiles.ProfileFileLocation
+import software.aws.toolkits.core.utils.debug
+import software.aws.toolkits.core.utils.getLogger
+import software.aws.toolkits.core.utils.info
+import java.nio.file.Paths
 
 class ProfileWatcher(parentDisposable: Disposable) : AsyncFileListener, Disposable {
     private val watchRoots = mutableSetOf<LocalFileSystem.WatchRequest>()
+    private val watchPointers = mutableMapOf<String, VirtualFilePointer>()
     private var onUpdate: (() -> Unit)? = null
 
     init {
         Disposer.register(parentDisposable, this)
     }
 
     private val watchLocationsStrings = setOf(
-        FileUtil.normalize(ProfileFileLocation.configurationFilePath().toAbsolutePath().toString()),
-        FileUtil.normalize(ProfileFileLocation.credentialsFilePath().toAbsolutePath().toString())
+        ProfileFileLocation.configurationFilePath().toAbsolutePath().toString(),
```

Comment:
Why are we no longer calling normalize?

Reply:
That was a mistake

Reply:
A mistake that we should never have called normalize?

Reply:
Mistake it was removed, it is back but in a different spot